packages/config/src/layer2s/aztecconnect.ts

```diff
@@ -61,6 +61,11 @@ export const aztecconnect: Layer2 = {
         sinceTimestamp: new UnixTime(1654638194),
       },
     ],
+    transactionApi: {
+      type: 'rpc',
```

Comment:
Why not have the type be `aztec`, then you wouldn't have to filter for those project ids specifically